### PR TITLE
Ubuntu 24.04 Upgrade

### PR DESCRIPTION
**Pull request by jfavellar90**
This PR aims to close https://github.com/overhangio/tutor-xqueue/issues/32

### CODE REVIEW

**Comment by DawoudSheraz** [comment]
@jfavellar90 Hi, what's the status of this PR? It has been in draft for a while. Thanks

**Comment by jfavellar90** [comment]
@DawoudSheraz it is still WIP since the Tutor core [PR](https://github.com/overhangio/tutor/pull/1117/files) is also WIP. Please confirm if I can jump and test Xqueue against the current state of the Tutor core PR.

**Comment by DawoudSheraz** [comment]
> @DawoudSheraz it is still WIP since the Tutor core [PR](https://github.com/overhangio/tutor/pull/1117/files) is also WIP. Please confirm if I can jump and test Xqueue against the current state of the Tutor core PR.

I think you can go ahead with testing and making this PR ready for review. You can use core PR for testing if needed.

File: tutorxqueue/templates/xqueue/build/xqueue/Dockerfile [hunk 1]
```diff
@@ -1,25 +1,23 @@
...
+        python3 \
+        python3-pip \
+        python3-venv \
+        libmysqlclient-dev \
+        pkg-config
+RUN ln -s /usr/bin/python3 /usr/bin/python
 
 ###### Git-clone xqueue repo ######
-ARG APP_USER_ID=1000
+ARG APP_USER_ID=1001
```
Comment (DawoudSheraz) [inline tutorxqueue/templates/xqueue/build/xqueue/Dockerfile]: Please take a look at https://github.com/overhangio/tutor/pull/1117#discussion_r1795117242 for context on this change.

Comment (jfavellar90) [reply tutorxqueue/templates/xqueue/build/xqueue/Dockerfile]: @DawoudSheraz addressed

**Comment by jfavellar90** [comment]
@DawoudSheraz After testing with the new tutor image with Ubuntu 22.04, the Xqueue service worked as expected.

**Review by DawoudSheraz — COMMENTED** [review]
Please add a changelog entry.

**Comment by jfavellar90** [comment]
> Please add a changelog entry.

Done @DawoudSheraz

**Review by DawoudSheraz — APPROVED** [review]

**Pull request merged**